mlflow/server/js/package.json

```diff
@@ -16,7 +16,10 @@
     "prettier:fix": "prettier . --write",
     "prettier:check": "prettier . --check",
     "i18n": "node scripts/extract-i18n.js",
-    "i18n:check": "yarn i18n --lint"
+    "i18n:check": "yarn i18n --lint",
+    "graphql-codegen": "cd ../../.. && python ./dev/proto_to_graphql/code_generator.py && cd mlflow/server/js && yarn graphql-codegen:clean && yarn graphql-codegen:base",
```

Comment:
These `cd`s might a bit sketchy, open for ideas

Reply:
Would `python ../../../dev/proto_to_graphql/code_generator.py` fail due to broken cwd when executing python part?
Alternatively you can replace `cd ../../.. && python ./dev/proto_to_graphql/code_generator.py` with invoking some custom `.sh` script, which will determine git root dir first and then will run python code. Might be an overkill, but will be robust and will work regardless of place where you execute it.

Reply:
Right, `code_generator.py` would break due to wrong paths on the schema file we are trying to find.

Yeah let me try the git root idea, thanks!